SUMMARY.md

```diff
@@ -137,6 +137,8 @@
 	* [StartRecording](voice/bxml/verbs/startRecording.md)
 	* [StopRecording](voice/bxml/verbs/stopRecording.md)
 	* [Transfer](voice/bxml/verbs/transfer.md)
+	    * [PhoneNumber](voice/bxml/verbs/transferPhoneNumber.md)
+	    * [Conference](voice/bxml/verbs/transferConference.md)
```

Comment:
I think for conference it makes sense to de-intent here to allow for it to appear on the drop down. Anecdotally i had to check here to find where we put the docs

Reply:
It feels weird to put it in the first level because it's not a verb, but at the same time I missed an easy-to-find reference to "how to conference". Maybe we should at least put something in "Guides & Tutorials".

Reply:
I put it in the first level with spaces before them. It looks as if they are nested but they are visible without clicking on `Tranfer`.